service/src/main/resources/api/service_openapi.yaml

```diff
@@ -145,6 +145,80 @@ paths:
         '500':
           $ref: '#/components/responses/ServerError'
 
+  /api/profiles/v1/{profileId}/policies:
+    parameters:
+      - $ref: '#/components/parameters/ProfileId'
+    get:
+      tags:
+        - Profile
+      summary: Retrieve policies for the profile
+      operationId: getProfilePolicies
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SamPolicyModelList'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '403':
+          $ref: '#/components/responses/PermissionDenied'
+        '404':
+          $ref: '#/components/responses/NotFound'
+
+  /api/profiles/v1/{profileId}/policies/{policyName}/members/{memberEmail}:
```

Comment:
The jade-data-repo version of this supplies the `memberEmail` through the request policy instead of having it in the URL (for POST, not DELETE). I don't see a compelling reason to do that though (unless we plan to bulk add users), and prefer to keep this method symmetric with the DELETE endpoint.